### PR TITLE
Moved event loop to Application class and fixed some brace styling.

### PR DESCRIPTION
**Pull request by FancyKillerPanda**
Event loop is now in Application class to keep it more self contained. Also made all braces start on a new line.

### CODE REVIEW

File: include/Application.hpp [hunk 1]
```diff
@@ -15,10 +17,13 @@ namespace ccm {
...
 
 		void render();
+		void handleEvents();
+
 		bool m_quit{ false };
 
 	private:
 		SDL_Window* m_window{ nullptr };
 		SDL_Renderer* m_renderer{ nullptr };
+		SDL_Event m_event;
```
Comment (Itay2805) [inline include/Application.hpp]: Do we really need the event to be part of the Application class?

Comment (FancyKillerPanda) [reply include/Application.hpp]: Shouldn't the Application class handle everything it needs? When the project is bigger, we wouldn't want to have the event loop in main as we'd have to be changing Application state, right?

Comment (Gaztin) [reply include/Application.hpp]: Yes but what itay means is that the SDL_Event variable can be local to the `handleEvents()` function

Comment (FancyKillerPanda) [reply include/Application.hpp]: I changed it

**Comment by Itay2805** [comment]
nice work!

**Pull request merged**